src/encryption.ts

```diff
@@ -183,21 +183,21 @@ export function decrypt({
         ciphertext,
         nonce,
         ephemPublicKey,
-        recieverEncryptionPrivateKey,
+        receiverEncryptionPrivateKey,
       );
 
       // return decrypted msg data
-      let output;
       try {
-        output = naclUtil.encodeUTF8(decryptedMessage);
+        if (!decryptedMessage) {
+          throw new Error();
+        }
+        return naclUtil.encodeUTF8(decryptedMessage);
       } catch (err) {
-        throw new Error('Decryption failed.');
-      }
-
-      if (output) {
```

Comment:
This had the effect of throwing the "Decryption failed." error if the resulting output was an empty string. Perhaps that's a behavior we should preserve in this PR at least (I'm not entirely sure why we'd want to throw in that circumstance but :shrug: )